adr/0032-no-assertion-test-triage-classification.md

```
@@ -0,0 +1,96 @@
+# ADR-0032: No-Assertion Test Triage Classification Framework
+
+## Status
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)